### PR TITLE
Disable content scrolling when offers dialog is visible

### PR DESCRIPTION
**Pull request by fikus**
Internal tracking: b/227460883

### CODE REVIEW

**Review by adolgachev — APPROVED** [review]

**Comment by fikus** [comment]
@jpettitt can you take a look, please?

File: src/runtime/contributions-flow.js [hunk 1]
```diff
@@ -73,6 +76,7 @@ export class ContributionsFlow {
                 'skus': (options && options.skus) || null,
                 'isClosable': isClosable,
                 'supportsEventManager': true,
+                'shouldDisableBodyScrolling': shouldDisableBodyScrolling,
```
Comment (hartmannr76) [inline src/runtime/contributions-flow.js:79]: I think these are args that are passed into the iframe to receive. Any reason this prob can't just be passed into as a `DialogConfig` setting? https://github.com/subscriptions-project/swg-js/blob/f56b586bd4cb92c443b2db1667e1cc677123d8c6/src/components/dialog.js#L69-L85
I imagine it'd reduce the amount of filed being touched for that change

Comment (fikus) [reply src/runtime/contributions-flow.js]: Updated. Thanks for the suggestion!

File: src/ui/activity-iframe-view.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export class ActivityIframeView extends View {
    * @param {!Window} win
    * @param {!../components/activities.ActivityPorts} activityPorts
    * @param {string} src
-   * @param {!Object<string, ?>=} args
+   * @param {!Object<string, ?>=} args Additional data to be passed to the iframe.
```
Comment (hartmannr76) [inline src/ui/activity-iframe-view.js:36]: +1 thank you for the comment

**Review by hartmannr76 — APPROVED** [review]

File: src/components/dialog.css [hunk 3]
```diff
@@ -90,3 +90,16 @@
     margin-left: 0 !important;
   }
 }
+
+/**
+ * Class applied to content page to disable scrolling.
+ */
+body.swg-disable-scroll {
+  margin: 0 !important;
```
Comment (jpettitt) [inline src/components/dialog.css:98]: Wondering if the margin 0 might screw up some sites (I know I suggested it but I'm not 100% sure it's needed to stop scrolling)

Comment (fikus) [reply src/components/dialog.css:98]: I'm also not sure if that is needed. I can remove it, to err on the side of being less intrusive.

Comment (jpettitt) [reply src/components/dialog.css:98]: Probably a good idea, we can always add it if needed later.

**Pull request merged**